### ISSUE TITLE
lat/lon coordinate

### ISSUE THREAD

**Issue by kyao** [usc-isi-i2/t2wml#138]
Getting the following error while generating kgtk edge file. Error occurs in interactive Windows executable and in version 19.

`ValueError: dict contains fields not in fieldnames: ' node2;kgtk:globe'`

Data is:
```
country,longitude,latitude,value
Ethiopia,40,9,100
USA,41,10,101
```

Yaml is:
```
statementMapping:
  region:
    - left: D
      right: D
      top: 2
      bottom: 3
  template:
    item: =item[A, $row]
    property: P1000
    value: =value[D, $row]
    qualifier:
      - property: P625
        longitude: =value[B, $row]
        latitude: =value[C, $row]
        globe: wgs84
```

Wikifier is:
```
column,row,value,context,item
,,Ethiopia,,Q115
,,USA,,Q30
```

**Comment by kyao** [comment]
I made a fix in version 19 of the code so that it generates correct kgtk edge files. I did not check in, the change, because in the current development branch the method kgtk_add_property_type_specific_fields() in mapping.py looks quite different.

Only two lines are changed:
```
25c25
<         result_dict["node2;kgtk:data_type"]="location_coordinates"
---
>         result_dict["node2;kgtk:data_type"]="coordinate" #not defined for sure yet
29c29
<         # result_dict[" node2;kgtk:globe"]=property_dict.get("globe", "")
---
>         result_dict[" node2;kgtk:globe"]=property_dict.get("globe", "")
```

**Comment by kyao** [comment]
Here is a list of kgtk data types:

```
$ kgtk implode --show-data-types
empty
list
number
quantity
string
language_qualified_string
location_coordinates
date_and_times
extension
boolean
symbol

**Comment by devowit** [comment]
fixed in 260fde7

**Comment by kyao** [comment]
Tested
